app/models/like.rb

```diff
@@ -0,0 +1,14 @@
+class Like < ApplicationRecord
+  # belongs_to :user
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Adding `foreign_key: 'author_id'` in this line of code may create some issues. Remember you used `user_id `instead of `author_id` in the likes table.